frontend/src/modules/machinery/components/TranslationSource.test.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import TranslationSource from './TranslationSource';
+
+
+const DEFAULT_TRANSLATION = {
+    sources: [
+        {
+            type: 'translation-memory',
+            url: 'http://pontoon.mozilla.org',
+        },
+    ]
+};
+
+
+describe('<TranslationSource>', () => {
+    [{id: 'translation-memory', component: 'TranslationMemory'},
+        {id: 'google-translate', component: 'GoogleTranslation'},
+        {id: 'microsoft-translator', component: 'MicrosoftTranslation'},
+        {id: 'microsoft-terminology', component: 'MicrosoftTerminology'},
+        {id: 'transvision', component: 'TransvisionMemory'},
+        {id: 'caighdean', component: 'CaighdeanTranslation'},
+    ].forEach(({id, component}) =>
```

Comment:
We use a tool to do just that: `jest-each`. See a usage example here: https://github.com/mozilla/pontoon/blob/master/frontend/src/core/placeable/parsers/camelCaseString.test.js